apps/api/src/controllers/languages.controller.test.ts

```
@@ -0,0 +1,537 @@
+
+  beforeEach(() => {
+    mockResJson.mockClear();
+    mockReqJson.mockClear();
+    mockReqParam.mockClear();
+    mockReqQuery.mockClear();
+
+    // Reset all db method mocks in mockDbInstance
+    Object.values(mockDbInstance).forEach(m => {
+      if (m && typeof m.mockClear === 'function') m.mockClear();
+      // For nested mocks like insert().values().returning()
+      if (m.mock && m.mock.results) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)